README.asciidoc

```diff
@@ -369,6 +369,19 @@ command names:
 ------------------------------------------------------------------------------------------
 
 
+[[Scripting]]
+Hints on scripted usage
+-----------------------
+If you want to run image-bootstrap from a shell script and capture its output
+to STDOUT/STDERR in a log file, be sure to run image-bootstrap in unbuffered
+mode, e.g.:
+
+`python -u image-bootstrap [params] 2>&1 | tee my.log`
+
+The default shebang of image-bootstrap does NOT use `-u`. Without it,
+STDOUT/STDERR won't be synchronized and error output you see in a log file will
+not exactly correspond to preceding/succeding output on STDOUT.
```

Comment:
I believe "succe[e]ding" is missing an `e`.

Reply:
ack